tests/www/views/test_session.py

```diff
@@ -98,3 +98,11 @@ def test_check_active_user(app, user_client):
     resp = user_client.get("/home")
     assert resp.status_code == 302
     assert "/logout" in resp.headers.get("Location")
+
+
+def test_check_deactivated_user_redirected_to_login(app, user_client):
+    user = app.appbuilder.sm.find_user(username="test_user")
+    user.active = False
+    resp = user_client.get("/home", follow_redirects=True)
+    assert resp.status_code == 200
+    assert '/login' in resp.request.url
```

Comment:
The test doesn't reproduce the problem. I tested it with the main branch, and it passed.

Reply:
Yes, I realized it as well after pushed a test. Thanks for spotting! I updated the test, now if you remove the added logic you will get this test failing with the following error:
`FAILED tests/www/views/test_session.py::test_check_deactivated_user_redirected_to_login - werkzeug.test.ClientRedirectError: Loop detected: A 302 redirect to /logout/ was already made.`

Please let me know if there is anything else needed.